Monika After Story/game/script-stories.rpy

```diff
@@ -348,11 +412,556 @@ label mas_story_immortal_love:
+    m 1esb "The two spent much of the night together dancing. When the time came to leave, Lewis offered Mary a ride home and she again accepted the invitation."
+    m 3esb "She directed him to drive down a certain road, and he obliged."
+    m 3eud "As they were passing the gates of a cemetery, Mary asked Lewis to pull over."
+    m 1eud "Although perplexed, Lewis stopped the car as she requested."
+    m 3eud "She then opened the door, leaned in towards Lewis, whispered that she had to go and that he could not go with her."
+    m 1euc "She then got out of the car and walked towards the cemetery gate before disappearing."
+    m "Lewis sat in the car for a long time bewildered by what had just happened."
+    m 1esd "He never saw the beautiful woman ever again."
+    # 1 in 20
+    if renpy.random.randint(1,20) == 1 or mas_full_scares:
+        play sound "sfx/giggle.ogg"
+    call mas_scary_story_cleanup
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_scary_story_corpse",
+    category=[store.mas_stories.TYPE_SCARY], prompt="The resuscitated corpse",unlocked=False),
+    eventdb=store.mas_stories.story_database)
+
+label mas_scary_story_corpse:
+    call mas_scary_story_setup
+    m 1esa "There once was an old man that ran an old roadside inn. One evening, 4 men arrived and asked for a room."
+    m 3eua "The old man replied that all of the rooms were taken, but he could find them a place to sleep if they weren’t too particular."
+    m 1esa "The men were exhausted and assured the old man that any place would do."
+    m 1eud "The old man led them to a room around back. Lying in the corner of the room was the corpse of a woman."
+    m "The old man explained that his daughter-in-law had recently perished and she was awaiting burial."
+    m 1eua "After the old man departed, 3 of the 4 men fell asleep. The last man couldn’t fall asleep."
+    m 1wuo "Suddenly, the man heard a creaking noise."
+    if renpy.random.randint(1,2) == 1 or mas_full_scares:
+        play sound "sfx/crack.ogg"
+    m 3wuo "He looked up and in the light of the lamp, he saw the woman rise, now bearing fangs and fingernails that looked like claws, advancing towards them."
+    m "She bent down and bit each of the sleeping men. The fourth man, at the last second, pulled up a pillow in front of his neck."
+    m 1eud "The woman bit the pillow and apparently not realizing she hadn’t bit the last man, returned to her original resting spot."
+    m 3eud "The man kicked his companions, but none of them moved. The man decided to take his chances and make a run for it."
+    m 3wuo "As soon as his feet touched the ground, however, he heard another creak."
+    m "Realizing that the woman was again rising from her spot, he opened the door and ran as fast as he could."
+    # need opinions on this one since it's for storytelling purposes
+    show layer master at heartbeat2(1)

```

Comment:
should also add the heartbeat sound here too

Reply:
📈